src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -0,0 +1,227 @@
+            DependencyUpdateErrorProcessorOptions = dependencyUpdateErrorProcessorOptions.Value;
+        }
+
+        public IReliableStateManager StateManager { get; }
+
+        public ILogger<DependencyUpdateErrorProcessor> Logger { get; }
+
+        public BuildAssetRegistryContext Context { get; }
+
+        public DependencyUpdateErrorProcessorOptions DependencyUpdateErrorProcessorOptions { get; }
+
+        [CronSchedule("0 0 0/1 1/1 * ? *", TimeZones.PST)]
+        public async Task DependencyUpdateErrorProcessing()
+        {
+
+            if (DependencyUpdateErrorProcessorOptions.ConfigurationRefresherdPointUri != null && DependencyUpdateErrorProcessorOptions.DynamicConfigs != null)
+            {
+                DependencyUpdateErrorProcessorOptions.ConfigurationRefresherdPointUri.Refresh().GetAwaiter().GetResult();
+           
+                bool.TryParse(DependencyUpdateErrorProcessorOptions.DynamicConfigs["FeatureManagement:DependencyUpdateErrorProcessor"], 
+                    out var autoDependencyUpdateErrorProcessor);
+                if (autoDependencyUpdateErrorProcessor)
+                {
+                    IReliableDictionary<string, DateTime> update =
+                        await StateManager.GetOrAddAsync<IReliableDictionary<string, DateTime>>("update");
+                    try
+                    {
+                        using (ITransaction tx = StateManager.CreateTransaction())
+                        {
+                            var previousTransactionDateTime = await update.GetOrAddAsync(
+                             tx,
+                             "update",
+                             DateTime.Now
+                             );
+                            var updatedTime = await CheckForError(previousTransactionDateTime, DependencyUpdateErrorProcessorOptions.GithubUrl);
+                            await update.TryUpdateAsync(
+                                tx,
+                                "update",
+                                updatedTime.UtcDateTime,
+                                previousTransactionDateTime
+                                );
+                            await tx.CommitAsync();
+                        }
+                    }
+                    catch (Exception ex)
+                    {
+                        Logger.LogError(ex, "Processing update dependency messages");
+                    }
+                }
+            }
+            else
+            {
+                Logger.LogInformation("Dependency Update Error processor is disabled because no App Configuration was available.");
+            }
+        }
+
+        public async Task<DateTimeOffset> CheckForError(DateTimeOffset previousTransactionDateTime , string whereToCreateIssue)
+        {
+            List<RepositoryBranchUpdateHistoryEntry> list = (from repo in Context.RepositoryBranchUpdateHistory
+                                                             where repo.Success == false
```

Comment:
These aren't really "repo" things in this in table.  They appear to be... historyEntries of some time?